app/grandchallenge/challenges/tasks.py

```diff
@@ -1,37 +1,55 @@
 from celery import shared_task
 from django.core.mail import mail_managers
+from django.db.models import Count, Max
 from requests import exceptions, get
 
 from grandchallenge.challenges.models import Challenge, ExternalChallenge
 from grandchallenge.evaluation.models import Evaluation
+from grandchallenge.profiles.admin import User
```

Comment:
See https://docs.djangoproject.com/en/3.2/topics/auth/customizing/#referencing-the-user-model - instead import `get_user_model` and then replace instances of `User` with `get_user_model()`.